app/views/account/videos/index.html.slim

```diff
@@ -0,0 +1,18 @@
+- set_meta_tags title: "All videos of #{@company.name}",
+                description: "All videos of #{@company.name} we processed"
+.content-wrapper
+  .content-header.row
+    .content-header-left.col-md-6.col-12.mb-2
+      .content-header-title.mb-0
+        h3 Videos
+      .row.breadcrumbs-top
+        .breadcrumb-wrapper.col-12
+          = render_breadcrumbs({ 'Dashboard' => account_root_path,
+                                 @company.name => account_company_path(@company),
+                                 'Videos' => nil})
+  .content-body
+    section.row
+      .col-md-12
+        .row
+          = render @videos
```

Comment:
specify partial which is used here, e.g.

- videos.each do....
  = render '/accoiunt/videos/video', video: video